apps/ledger-live-desktop/src/renderer/screens/manager/DeviceDashboard/AppsList/Item.tsx

```diff
@@ -101,29 +126,47 @@ const Item = ({
     [app.bytes, availableApp?.bytes, deviceInfo.version, deviceModel, installed, onlyUpdate],
   );
 
+  const { enabled: displayAppDeveloperName } = useFeature("myLedgerDisplayAppDeveloperName") || {};
```

Comment:
`|| {};` needed?

Reply:
Didn't see previous comment 😅

Reply:
yes it's needed because useFeature can return `null`